### ISSUE TITLE
Easy Fix?

### ISSUE THREAD

**Issue by ghost** [Jessecar96/SteamBot#272]
Hey Guys, It's me again. sorry I keep asking about the SteamBot but I just really like it :). So my question is how can I edit this source code made by @waylaidwanderer (Thanks Again) ([from here](http://www.reddit.com/r/tf2trade/comments/19zvu2/psa_remember_when_i_released_the_source_code_for/)) so instead of weapons 2 for 1 scrap each but 1 hat for 1.33 each? It will still have all the same attributes as the weapons, so if it is not craftable, then they cant trade it to us, so basically just the bot formed to hats or items. 

So in summary, I want to edit @waylaidwanderer's code to bank hats and for it to bank weapons.  Thanks For The Support From The Community!
